README.md

```diff
@@ -67,15 +68,14 @@ openebs-csi-node-56t5g     2/2     Running   0          6m13s
 ### Provision a cStor volume using OpenEBS CSI driver
 
 1. Make sure you already have a cStor Pool Created or you can 
-   create one using the below command. In the below spc.yaml make sure 
-   that maxPools should be greater than or equal to the number of 
-   replicas required for the volume.
+   create one using the below command. In the below cspc.yaml make sure 
+   that the specified pools list should be greater than or equal to
+   the number of replicas required for the volume.
 
-   The following command will create the specified number of cStor Pools
-   using the Sparse files. 
+   The following command will create the specified cStor Pools in the cspc yaml:
 
    ```
-   kubectl apply -f https://raw.githubusercontent.com/openebs/csi/master/deploy/spc.yaml
+   kubectl apply -f https://raw.githubusercontent.com/openebs/csi/master/deploy/cspc.yaml
```

Comment:
this file needs to be added.

Reply:
This file has been added in a separate PR https://github.com/openebs/csi/pull/34

Reply:
I don't see cspc.yaml yet in that PR.

Reply:
added it now, had missed pushing the file.